zepid/causal/ipw/IPMW.py

```diff
@@ -79,11 +95,89 @@ def fit(self):
         print_results:
             -whether to print the model results. Default is True
         """
-        if self.denominator_model is False:
+        if self._denominator_model is False:
             raise ValueError('No model has been fit to generated predicted probabilities')
 
+        w = self.df['__numer__'] / self.df['__denom__']
+        self.Weight = w
+
+    def _check_monotone(self):
+        """Background function to check whether data is actually monotone. Raise ValueError if user requests
+        monotone=True, but the missing data is nonmonotonic.
+        """
+        miss_order = list(reversed(self.missing))
+        for m in range(len(miss_order)):
+            if m == 0:
```

Comment:
I think this is nicer (personal pref)

```python
if m==0 or miss_order[m] == len(miss_order):
    continue

post = np.where(self.df[miss_order[m]].isnull(), 1, 0)
prior = np.where(self.df[miss_order[m-1]].isnull(), 1, 0)
...

Reply:
Somehow I missed the `continue` functionality when learning loops. Thanks!